--- symlinks-1.2/symlinks.8.fixman	Tue May 30 14:49:29 2000
+++ symlinks-1.2/symlinks.8	Tue May 30 14:50:43 2000
@@ -1,11 +1,11 @@
-.TH SYMLINKS 8 "November 1994" "Version 1.1"
+.TH SYMLINKS 8 "November 1994" "Version 1.2"
 
 .SH NAME
 symlinks \- symbolic link maintenance utility
 .SH SYNOPSIS
 .B symlinks
 [
-.B -cdrv
+.B -cdrstv
 ]
 dirlist
 .SH DESCRIPTION
--- symlinks-1.2/symlinks.c.fixman	Tue May 30 14:59:59 2000
+++ symlinks-1.2/symlinks.c	Tue May 30 15:00:39 2000
@@ -289,7 +289,7 @@
 	fprintf(stderr, "Flags:\t-c == change absolute/messy links to relative\n"
 		"\t-d == delete dangling links\n"
 		"\t-r == recurse into subdirs\n"
-		"\t-s == shorten lengthy links\n"
+		"\t-s == shorten lengthy links (only displayed if -c not specified)\n"
 		"\t-v == verbose (show all symlinks)\n\n");
 	exit(1);
 }
